src/context/DataPlaneScopeContext.tsx

```diff
@@ -1,21 +1,33 @@
 import { createContext, useCallback, useContext, useState } from 'react';
+import { useLocalStorage } from 'react-use';
 import { DataPlaneScopes } from 'stores/DetailsForm/types';
 import { BaseComponentProps } from 'types';
+import { LocalStorageKeys } from 'utils/localStorage-utils';
 
 interface DataPlaneScope {
     dataPlaneScope: DataPlaneScopes;
     toggleScope: Function;
 }
 
+const defaultOption: DataPlaneScopes = 'private';
 const DataPlaneScopeContext = createContext<DataPlaneScope | null>(null);
 
 const DataPlaneScopeContextProvider = ({ children }: BaseComponentProps) => {
-    const [dataPlaneScope, setDataPlaneScope] =
-        useState<DataPlaneScopes>('private');
+    const [defaultDataPlane, setDefaultDataPlane] =
+        useLocalStorage<DataPlaneScopes>(
+            LocalStorageKeys.ADMIN_DATAPLANE_CHOICE,
+            defaultOption
+        );
```

Comment:
Trying to make the UX better and not make users have to constantly switch between public and private. Eventually - this should be a setting stored with the select tenant.